src/main/java/org/fundacionjala/coding/nestor/NewReleasePrice.java

```diff
@@ -0,0 +1,13 @@
+package org.fundacionjala.coding.nestor;
+
+public class NewReleasePrice extends Price {
+    @Override
+    MovieCode getPriceCode() {
+        return MovieCode.NEW_RELEASE;
+    }
+
+    @Override
+    public double getCharge(int daysRented) {
+        return daysRented * 3;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Cast one of the operands of this multiplication operation to a "double". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2184)